UI/MapUI.gd

```diff
@@ -31,7 +33,12 @@ func _on_return_button_press() -> void:
 	
 
 func _on_add_torch_pressed() -> void:
-	torch_confirmation_dialog.show()
+	if !PlayerManager.is_player_initial_position_set:
+		cant_set_torch_dialog.dialog_text = player_position_not_set_text
+		cant_set_torch_dialog.show()
+	#TODO: Add in torch count here
```

Comment:
This should be done later, i was thinking it should either be a player property, or an inventory property (which doesn't exist yet but would be used to put things like money, torches and items)